wordpress-mobile/gutenberg-mobile
# Release/update develop with 1.25.0 master

Update develop branch with release 1.25 from master

To test:
 - Check if the demo apps are working ok
 - Check dark mode in Android is correct

PR submission checklist:
